### DISCUSSION TITLE
无法翻译，页面会显示各种红色感叹号

### DISCUSSION THREAD

**Discussion by xiaochouyu2508** [immersive-translate/immersive-translate#482] [Help]
### 概述 | Summary

点击感叹号会显示 invade client ip
![1684736516347](https://github.com/immersive-translate/immersive-translate/assets/134259712/de9be3a7-8733-4a8f-aa3f-d0ac392bb069)

**Comment by theowenyoung** [comment]
似乎是你的ip被封禁了。换个ip或者换个翻译服务吧。
